<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyze Output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2">
   <item>
    <widget class="QLabel">
     <property name="text">
      <string>Blocks Found:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="blockOutputTableView">
    </widget>
   </item>
   <item>
    <widget class="QLabel">
     <property name="text">
      <string>Entities Found:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="entityOutputTableView">
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="txtButton">
       <property name="text">
        <string>Export txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="csvButton">
       <property name="text">
        <string>Export csv</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
<connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
